models/cloud-service-providers.ts

```diff
@@ -0,0 +1,30 @@
+export enum Network {
+  NETWORK_1 = "NETWORK_1",
+  NETWORK_2 = "NETWORK_2",
+  NETWORK_3 = "NETWORK_3",
+}
+export interface ICloudServiceProvider {
+  name: string;
+  uri: string;
+  networks: Array<Network>;
+}
+export class CloudServiceProvider implements ICloudServiceProvider {
```

Comment:
I wondered if they were needed, since the `CloudServiceProvider` defines them as `readonly`. I do not see a reason _not_ to add them though, so adding them.